calcom/cal.com
# fix: add team bio to public page

## What does this PR do?

Currently on the public page for a team, the team's bio is not presented, This PR adds that bio in. 
This PR also replaces the styling for the Team Name to match the one on the single user page. Let me know if this should be reversed. 

(Apologies for massive images again)

This is the team page currently.
![2021-12-07-104247_1266x885_scrot](https://user-images.githubusercontent.com/10099540/145014754-d8dac890-d015-401d-b208-942974c47912.png)

This is the team page afterwards.
![2021-12-07-104034_1168x1138_scrot](https://user-images.githubusercontent.com/10099540/145014784-2853c9f8-8bbe-4044-a215-e803f768c51a.png)

For Reference this is a single user page.
![2021-12-07-104019_1172x998_scrot](https://user-images.githubusercontent.com/10099540/145014810-2822db60-c737-4df7-9200-1321f0678fca.png)

Also note I added the logo for the team without the mimetype changes in these fixes which is why it looks different.


## Type of change


## How should this be tested?

Add a bio to your team and navigate to your teams public page. 
It should now be visible for the world to learn more about your team.
As always happy to add some tests for this, but I'm not sure regarding the testing strategy of this project.

## Checklist:
